pkg/builder/action_merge_meta_data_builder.go

```diff
@@ -190,47 +200,19 @@ func (rb *ResizeMergePolicyBuilder) Build() (*proto.ActionMergePolicyDTO, error)
 
 // Horizontal Scale Merge Policy DTO builder
 type HorizontalScaleMergePolicyBuilder struct {
-	entityType                *proto.EntityDTO_EntityType
-	chainedAggregationTargets []*ActionDeDuplicateAndAggregationTargetBuilder
-	commTypes                 []*CommodityMergeData
-	entityFilters             []*proto.EntityDTO_ContainerPodData
-}
-
-func NewHorizontalScaleMergeSpecBuilder() *HorizontalScaleMergePolicyBuilder {
-	return &HorizontalScaleMergePolicyBuilder{}
-}
-
-func (hsb *HorizontalScaleMergePolicyBuilder) ForEntityType(entityType proto.EntityDTO_EntityType) *HorizontalScaleMergePolicyBuilder {
-	hsb.entityType = &entityType
-	return hsb
+	*MergePolicyBuilder
+	entityFilters []*proto.EntityDTO_ContainerPodData
```

Comment:
I think the entity filters should be in the "base" `MergePolicyBuilder` struct since it can be used for all merge policies, not just horizontal scale. You may also want to rename this property to `entityExclusionFilters` since I was asked to name mine that way.

Reply:
Yes, renaming to entityExclusionFilters is a good idea! But what should be the type of this filters if we want to that to be in the base MergePolicyBuilder struct?

Reply:
Wouldn't the type be `EntityFilter` since that's what you put into the [proto](https://github.ibm.com/turbonomic/opsmgr/blob/staging/com.vmturbo.platform.sdk.common/src/main/protobuf/ActionExecution.proto#L124)?

Reply:
using this? 
entityFilters             []interface{}

Reply:
You may be right? I will try that out.

Reply:
Thanks! I did more refactoring based on your suggestion.